dev/archery/archery/cli.py

```diff
@@ -643,5 +647,78 @@ def trigger_bot(event_name, event_payload, arrow_token, crossbow_token):
     bot.handle(event_name, event_payload)
 
 
+@archery.group('docker')
+@click.option('--config', '-c', type=click.Path(exists=True),
```

Comment:
Use `--src` like the other commands. See [archery/utils/source.py,](https://github.com/apache/arrow/blob/master/dev/archery/archery/utils/source.py) you may want to add a `docker_compose` property.